docs/information/file-structure.md

```diff
@@ -40,8 +40,8 @@ The optional `assets` folder can contain icons that will be packaged into the ex
 
 The directory contains a few more files that setup common JavaScript tooling:
 
-- **eslint.config.js** describes rules for [ESLint](https://eslint.org), which you can run with `npm run lint`. It has recommendations for code style and best practices. Usually, you don't have to edit this file.
 - **.prettierrc** contains default rules for [Prettier](https://prettier.io) to format your code. We recommend to setup the [VS Code extension](https://prettier.io/docs/en/editors.html#visual-studio-code) to keep your code pretty automatically.
+- **eslint.config.mjs** describes rules for [ESLint](https://eslint.org), which you can run with `npm run lint`. It has recommendations for code style and best practices. Usually, you don't have to edit this file.
```

Comment:
it doesn't have to be mjs, js is fine

Reply:
In the `eslint.md` documentation, we're using the ESM syntax for importing and exporting modules.

The `.mjs` is also more accurate for our existing ESM extensions to migrate. Using `.js` extension means the file follows the configuration in the `package.json` (CommonJS or ESM). And using `.mjs` will not cause any misleading to our developers.

What do you think?

Reply:
I hate .mjs 😅 

I'd say let's stick to `.js`, it's one less concept to look at. If people want to use `.mjs`, they'll know how.

So maybe let's update eslint.md? I didn't catch it earlier

Reply:
Sure

Reply:
Please let me know if you want all `eslint.config.mjs` files in examples to be `.js`.